charts/tidb-cluster/templates/monitor-deployment.yaml

```diff
@@ -134,6 +134,11 @@ spec:
           - name: prometheus-rules
             mountPath: /prometheus-rules
             readOnly: false
+          {{- if .Values.enableTLSCluster }}
+          - name: tls-pd-client
```

Comment:
What about tidb?

Reply:
The same cert pair can be used for every endpoint, as long as the cert is signed by the same CA as server, using any one of our valid secret would work.

Reply:
OK, got it.